test/convnets.jl

```diff
@@ -6,177 +6,201 @@ PRETRAINED_MODELS = []
 
-    @test size(m(rand(Float32, 224, 224, 3, 1))) == (1000, 1)
+    @test size(m(x_224)) == (1000, 1)
     if (VGG, sz, bn) in PRETRAINED_MODELS
       @test (VGG(sz, batchnorm = bn, pretrain = true); true)
     else
       @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
     end
-    @test_skip gradtest(m, rand(Float32, 224, 224, 3, 1))
+    @test gradtest(m, x_224)
+    GC.safepoint()
+    GC.gc()
   end
 end
 
+GC.safepoint()
 GC.gc()
 
 @testset "ResNet" begin
   @testset "ResNet($sz)" for sz in [18, 34, 50, 101, 152]
     m = ResNet(sz)
 
-    @test size(m(rand(Float32, 256, 256, 3, 1))) == (1000, 1)
+    @test size(m(x_256)) == (1000, 1)
     if (ResNet, sz) in PRETRAINED_MODELS
       @test (ResNet(sz, pretrain = true); true)
     else
       @test_throws ArgumentError ResNet(sz, pretrain = true)
     end
-    @test_skip gradtest(m, rand(Float32, 256, 256, 3, 2))
+    @test gradtest(m, x_256)
+    GC.safepoint()
+    GC.gc()
   end
 
   @testset "Shortcut C" begin
     m = Metalhead.resnet(Metalhead.basicblock, :C;
                          channel_config = [1, 1],
                          block_config = [2, 2, 2, 2])
 
-    @test size(m(rand(Float32, 256, 256, 3, 1))) == (1000, 1)
+    @test size(m(x_256)) == (1000, 1)
+    @test gradtest(m, x_256)
   end
 end
 
+GC.safepoint()
 GC.gc()
 
 @testset "ResNeXt" begin
   @testset for depth in [50, 101, 152]
     m = ResNeXt(depth)
 
-    @test size(m(rand(Float32, 224, 224, 3, 1))) == (1000, 1)
+    @test size(m(x_224)) == (1000, 1)
     if ResNeXt in PRETRAINED_MODELS
       @test (ResNeXt(depth, pretrain = true); true)
     else
       @test_throws ArgumentError ResNeXt(depth, pretrain = true)
     end
-    @test_skip gradtest(m, rand(Float32, 224, 224, 3, 2))
+    @test gradtest(m, x_224)
+    GC.safepoint()
+    GC.gc()
   end
 end
 
+GC.safepoint()
 GC.gc()
 
 @testset "GoogLeNet" begin
   m = GoogLeNet()
-  @test size(m(rand(Float32, 224, 224, 3, 1))) == (1000, 1)
+  @test size(m(x_224)) == (1000, 1)
   @test_throws ArgumentError (GoogLeNet(pretrain = true); true)
-  @test_skip gradtest(m, rand(Float32, 224, 224, 3, 1))
+  @test gradtest(m, x_224)
 end
 
+GC.safepoint()
 GC.gc()
 
 @testset "Inception3" begin
   m = Inception3()
-  @test size(m(rand(Float32, 224, 224, 3, 1))) == (1000, 1)
+  @test size(m(x_224)) == (1000, 1)
   @test_throws ArgumentError Inception3(pretrain = true)
-  @test_skip gradtest(m, rand(Float32, 224, 224, 3, 2))
+  @test gradtest(m, x_224)
 end
 
+GC.safepoint()
 GC.gc()
 
 @testset "SqueezeNet" begin
   m = SqueezeNet()
-  @test size(m(rand(Float32, 224, 224, 3, 1))) == (1000, 1)
+  @test size(m(x_224)) == (1000, 1)
   @test_throws ArgumentError (SqueezeNet(pretrain = true); true)
-  @test_skip gradtest(m, rand(Float32, 224, 224, 3, 1))
+  @test gradtest(m, x_224)
 end
 
+GC.safepoint()
 GC.gc()
 
 @testset "DenseNet" begin
-  @testset for sz in [121, 161, 169, 201]
+  @testset for sz in [121, 169, 169, 201]
```

Comment:
Was this intentional?

Reply:
Oh I was trying something that didn't work out, it should be 161. I'll fix that